src/custom/state/price/reducer.ts

```diff
@@ -149,4 +149,10 @@ export default createReducer(initialState, (builder) =>
       // Stop the loader
       state.loading = false
     })
+    /**
+     * Signal only loading
+     */
+    .addCase(setQuoteLoading, (state, action) => {
```

Comment:
Isn't this action creating race conditions with getNewQuote, refreshQuote, updateQuote, setQuoteError?

Reply:
will investigate this further..